hathor/transaction/storage/transaction_storage.py

```diff
@@ -749,6 +749,13 @@ def update_last_started_at(self, timestamp: int) -> None:
                 continue
             self.set_index_last_started_at(index_db_name, timestamp)
 
+    def flush(self) -> None:
```

Comment:
This should probably be annotated with `@abstractmethod` so we don't forget to implement it.